regolith/fsclient.py

```diff
@@ -93,23 +95,42 @@ def load_json(self, db, dbpath):
             base, ext = os.path.splitext(collfilename)
             self._collfiletypes[base] = 'json'
             print('loading ' + f + '...', file=sys.stderr)
-            dbs[db['name']][base] = load_json(f)
+            coll = load_json(f)
+            dbs[db['name']][base] = coll
+            if base in self.mega_db:
+                for k, v in coll.items():
+                    if k in self.mega_db[base]:
+                        self.mega_db[base][k].update(v)
+                    else:
+                        self.mega_db[base][k] = v
+            else:
+                self.mega_db[base] = coll
 
     def load_yaml(self, db, dbpath):
         """Loads the YAML part of a database."""
         dbs = self.dbs
-        for f in iglob(os.path.join(dbpath, '*.y?ml')):
+        for f in iglob(os.path.join(dbpath, '*.y*ml')):
```

Comment:
Shouldn't this be a `?`. The only valid option is an `a`, which may or may not be present.  We shouldn't be matching thins like yajsjhsml

Reply:
The `?` failed on my machine with `.yml` files
"A '?' (not between brackets) matches any single character." So it won't match an empty character.